common/dynamicconfig/constants.go

```diff
@@ -1440,6 +1440,13 @@ const (
 	IsolationGroupStateUpdateRetryAttempts
 
 	LargeShardHistoryBlobMetricThreshold
+
+	// DeleteHistoryEventContextTimeout
```

Comment:
let's mention the unit of time in comment or variable name.

Reply:
Thanks, I added that in L3803 but forgot to add it here.